src/bios_handler.cpp

```diff
@@ -460,4 +463,93 @@ void IbmBiosHandler::processClearNvram()
     }
     logging::logMessage("Invalid type recieved for clear NVRAM from VPD.");
 }
+
+void IbmBiosHandler::saveKeepAndClearToVpd(const std::string& i_KeepAndClearVal)
+{
+    if (i_KeepAndClearVal.empty())
+    {
+        logging::logMessage(
+            "Empty value received for keep and clear from BIOS. Skip updating to VPD.");
+        return;
+    }
+
```

Comment:
should we check i_KeepAndClearVal size is 1, if not empty  ?

Reply:
Since the value is fetched at (0), I don't care much about what is there in rest of the string.

Reply:
I misunderstood with saveKeepAndClearToBios(), as there is check for size in that API.
here we checking with string 'enabled'.